packages/react/src/primitives/Alert/Alert.tsx

```diff
@@ -36,40 +36,38 @@ const AlertPrimitive: Primitive<AlertProps, typeof Flex> = (
     }
   }, [setDismissed, onDismiss, dismissed]);
 
-  return (
-    !dismissed && (
```

Comment:
TypeScript didn't like this because it means Alert can return `false`, which is not a valid ReactElement